typescript/src/services/tbtc.ts

```diff
@@ -74,6 +74,19 @@ export class TBTC {
     return TBTC.initializeEthereum(signer, "goerli", BitcoinNetwork.Testnet)
   }
 
+  /**
+   * Initializes the tBTC v2 SDK entrypoint for Ethereum Sepolia and Bitcoin testnet.
+   * The initialized instance uses default Electrum servers to interact
+   * with Bitcoin testnet
```

Comment:
Non-blocking, but the way we wrap those comments looks odd. The first line does not break at 80 characters and the second line breaks too early 🤔 I've got a margin set in my IDE at 80 characters and you can see it clearer on the image below:

<img width="799" alt="image" src="https://github.com/keep-network/tbtc-v2/assets/40306834/dfddfef6-be63-4236-9fcf-7c353dd73414">

I think the ideal scenario would be to keep the margin for comments at 80 characters, just like it is for the code:

<img width="653" alt="image" src="https://github.com/keep-network/tbtc-v2/assets/40306834/23e2a18c-400a-4d38-80b9-6682d7dcb18d">

The same thing is happening for `initializeEthereum` and `initializeGoerli` functions docs.

Just pointing that out. I'm not gonna block the merge because of that 😅